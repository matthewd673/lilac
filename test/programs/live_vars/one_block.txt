[BB_00]
i32 $a = 2i32
