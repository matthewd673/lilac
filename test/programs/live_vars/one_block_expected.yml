- id: ENTRY
  GEN: []
  KILL: []
  IN: []
  OUT: []
- id: BB_0
  GEN: []
  KILL:
    - a
  IN: []
  OUT: []
- id: EXIT
  GEN: []
  KILL: []
  IN: []
  OUT: []
